### ISSUE TITLE
Mailer README doesn't include .env instructions

### ISSUE THREAD

**Issue by j-norwood-young** [remp2020/remp#19]
The Mailer documentation is self-standing and complete except that it doesn't include the instructions for the .env file.

**Comment by rootpd** [comment]
Aren't the comments within [`.env.example`](https://github.com/remp2020/remp/blob/master/Mailer/.env.example) explanatory enough? We didn't want to duplicate that information also to README, it would get inconsistent over time.

**Comment by rootpd** [comment]
Maybe we could add the information about `.env.example` into the README? Would that help you when you read it before?

(maybe somewhere right at the top of readme and everywhere `.env` is mentioned)

**Comment by j-norwood-young** [comment]
Yeah, that would help tremendously. Just when working through the installation instructions, I hit the problem. It wasn't difficult to solve, it just felt like the instructions were incomplete.
